android/.gitignore

```diff
@@ -7,7 +7,7 @@ gradle-wrapper.jar
 GeneratedPluginRegistrant.java
 
 # Remember to never publicly share your keystore.
-# See https://flutter.dev/docs/deployment/android#reference-the-keystore-from-the-app
+# See https://flutter.dev/to/reference-keystore
 key.properties
 **/*.keystore
-**/*.jks
+**/*.jks
```

Comment:
Should have trailing newline. 
https://github.com/flutter/flutter/blob/71fd800e7e2163fbed7889c03830a20e1ba71d21/packages/flutter_tools/templates/app_shared/android.tmpl/.gitignore
Please apply to other files also. 

If you `cat` a file, you'll see that the `$` will be on the same line for files without trailing newline.

Reply:
please check it